cypress/integration/v2/features/dynamic-content/site-data.spec.js

```diff
@@ -0,0 +1,132 @@
+
+/**
+ * External dependencies
+ */
+import { registerTests } from '~stackable-e2e/helpers'
+
+describe( 'Dynamic Content - Site', registerTests( [
+	matchSiteData,
+	adjustFieldOptions,
+	adjustFieldValues,
+] ) )
+
+const fields = [ 'Site Title', 'Site Tagline', 'Site URL' ]
+const testString = [ 'Test Site Title', 'Test Site Tagline', 'Test Site URL' ]
```

Comment:
it might be better to restructure this.
we can do this so we can assert the values directly in assertBlockContent:

```
const fields = { [
   {
      name: 'Site Title',
      value: 'Test Site Title',
   },
   {
      name: 'Site Tagline',
      value: 'Test Site Tagline',
   },

...

] }